sdk/python/feast/usage.py

```diff
@@ -77,8 +77,8 @@ def log(self, function_name: str):
         self.check_env_and_configure()
         if self._usage_enabled and self.usage_id:
             if function_name == "get_online_features":
+                self._usage_counter["get_online_features"] += 1
                 if self._usage_counter["get_online_features"] % 10000 != 0:
-                    self._usage_counter["get_online_features"] += 1
```

Comment:
Nit, may be good to reset to 0 here to avoid the possibility of integer overflows. If things wrap around I guess that'd be fine too but we may as well take a little extra care

Reply:
May also be better to use `% 10000 !=` some higher number, like `% 10000 != 1`. The process that loads the feature store may be serializing and deserializing the feature store object, or creating new ones using different workers. If the counter doesnt work then it may log constantly.

Reply:
@achals that makes sense.

Reply:
@woop I think the current implementation does what you want: the counter is incremented to 1 to start, so the first logging will happen on the 10000th call, not on the first call.

Reply:
Alright.